modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/pojo.mustache

```diff
@@ -86,7 +86,8 @@ public class {{classname}} {{#parent}}extends {{{.}}} {{/parent}}{{#vendorExtens
     {{/isContainer}}
   {{/isXmlAttribute}}
   {{/withXml}}
-  {{#gson}}
+  {{#gson}}{{#deprecated}}
+  @Deprecated{{/deprecated}}
```

Comment:
Can you please use the following instead?
```
  {{#gson}}
  {{#deprecated}}
  @Deprecated
  {{/deprecated}}
```
That should avoid the blank line.

Same for line 142.

Reply:
Done